File: Iniciante/1010 - Cálculo Simples/JavaScript/1010.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const [idPart1, amountPart1, pricePart1] = lines[0].split(" ").map(Number);
  const [idPart2, amountPart2, pricePart2] = lines[1].split(" ").map(Number);
  const totalValue = amountPart1 * pricePart1 + amountPart2 * pricePart2;
  console.log(`VALOR A PAGAR: R$ ${totalValue.toFixed(2)}`);
}

main();

File: Iniciante/1010 - Cálculo Simples/Python/1010.py
def main():
    id_part_1_str, amount_part_1_str, price_part_1_str = input().split()
    id_part_2_str, amount_part_2_str, price_part_2_str = input().split()
    id_part_1 = int(id_part_1_str)
    amount_part_1 = int(amount_part_1_str)
    price_part_1 = float(price_part_1_str)
    id_part_2 = int(id_part_2_str)
    amount_part_2 = int(amount_part_2_str)
    price_part_2 = float(price_part_2_str)
    total_value = (amount_part_1 * price_part_1) + (amount_part_2 * price_part_2)
    print(f"VALOR A PAGAR: R$ {total_value:.2f}")


if __name__ == "__main__":
    main()
